src/protocols/at-ps2/keyboard_interface.c

```
@@ -642,7 +601,9 @@ static void handle_init_detection(void) {
     update_keyboard_ready_led(keyboard_state == INITIALISED);
 }
 
-void keyboard_interface_task() {
+// --- Public Functions ---
+
+void keyboard_interface_task(void) {
```

Cppcheck (2.19.0)

[style] The function 'keyboard_interface_task' is never used.

(unusedFunction)